### DISCUSSION TITLE
Retorno de status 404 e 403 das chamadas de obtenção da NF-e

### DISCUSSION THREAD

**Discussion by VictorMassayuki** [rodurma/forum-meli#31] [Geral]
Olá, estou validando os serviços para obter a nota fiscal no Postman, passando meu access token na header e todas as chamadas estou recebendo o status 404, conforme o print abaixo:

![image](https://github.com/rodurma/forum-meli/assets/48463621/1668b5ec-455b-490a-a2dd-acd7f028bb63)

E quando utilizo o usuário de teste, estou recebendo o status 403, conforme o print abaixo:

![image](https://github.com/rodurma/forum-meli/assets/48463621/bb47f40c-aa37-4a75-8091-9d581d2c7293)

Gostaria de saber se alguém passou por isso e como resolver.

**Comment by tegila** [comment]
```
{
  blocked_by: 'PolicyAgent',
  path: '/items?access_token=...&caller.detached.id=...&caller.id=...&caller.scopes=read%2Cwrite%2Coffline_access&caller.siteId=MLB&caller.status=ACTIVE&client.id=...&client.owner=...&root.id=...',
  status: 403,
  message: 'At least one policy returned UNAUTHORIZED.',
  code: 'PA_UNAUTHORIZED_RESULT_FROM_POLICIES'
}
```

Estou com o mesmo problema em um endpoint diferente. 
Já conseguiu descobrir o que é esse novo PolicyAgent ?
